plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeConnectorSmokeTest.java

```diff
@@ -2133,6 +2133,25 @@ public void testProjectionPushdownMultipleRows()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testPartitionFilterIncluded()
```

Comment:
Can we handle for cases where complex filter is applied on the partition column ?

Reply:
wip